frontend/public/components/RBAC/edit-rule.jsx

```diff
@@ -430,16 +437,18 @@ const EditRule = connect(stateToProps, { getResources: k8sActions.getResources }
                       activeValue={resourceControl}
                       onChange={this.set}
                       value={RESOURCE_ENUM.NON}
-                      label="Non-resource URLs"
-                      text="API URLs that do not correspond to objects"
+                      label={t('edit-rule~Non-resource URLs')}
+                      text={t('edit-rule~API URLs that do not correspond to objects')}
                     />
                     <HelpText>
                       <input
                         type="text"
                         value={nonResourceURLs}
                         className="pf-c-form-control text-input"
                         onChange={(e) => this.setNonResourceURL(e.target.value)}
-                        placeholder="Comma separated list of non-resource urls (/apis/extensions/v1beta1)"
+                        placeholder={t(
+                          'edit-rule~Comma separated list of non-resource urls (/apis/extensions/v1beta1)',
```

Comment:
Not new, but use URL (uppercase)

```suggestion
                          'edit-rule~Comma separated list of non-resource URLs (/apis/extensions/v1beta1)',
```

Reply:
Done